docs/content/node-operation/access-node-setup.mdx

```diff
@@ -1,22 +1,164 @@
+./bootstrap/
+├── private-root-information
+│        └── private-node-info_e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5
+│        ├── node-info.priv.json
+│        └── secretsdb-key
+└── public-root-information
+    ├── node-id
+    └── node-info.pub.e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5.json
+
+3 directories, 4 files
+```
+
+Your node identity has now been generated. Your <b>node ID</b> can be found in the file `./bootstrap/public-root-information/node-id`.
+
+```shell:title=Example
+$cat ./bootstrap/public-root-information/node-id
+e737ec6efbd26ef43bf676911cdc5a11ba15fc6562d05413e6589fccdd6c06d5
+```
+
+<Callout type="warning">
+    All your private keys should be in the `bootstrap` folder created earlier. Please take a back up of the entire folder.
+</Callout>
+
+
+## Step  2 - Stake the node
+
+You need to now register the node on chain by staking the node via [Flow Port](https://port.onflow.org/). [Here](https://developers.flow.com/nodes/flow-port) is a guide on how to use Flow port if you are not familiar with it.
+If you are staking via a custody provider or would like to directly submit a staking transaction instead follow this [guide](https://developers.flow.com/nodes/staking#how-do-i-stake).
+
+
+The `node details` (`Node ID`, `Network Address`, `Networking Key` and `Staking Key`) that need to be submitted when staking the node on Flow Port, can be found in the file: `./bootstrap/public-root-information/node-info.pub.<node-id>.json`.
+
+```shell:title=Example
+$cat ./bootstrap/public-root-information/node-info.pub.39fa54984b8eaa463e129919464f61c8cec3a4389478df79c44eb9bfbf30799a.json
+{
+  "Role": "consensus",
+  "Address": "consensus-001.nodes.flow.com:3569",
+  "NodeID": "39fa54984b8eaa463e129919464f61c8cec3a4389478df79c44eb9bfbf30799a",
+  "Weight": 1000,
+  "NetworkPubKey": "d92e3d5880abe233cf9fe9104db34bbb31251468a541454722b3870c04156a1b0504aef443bcaad124b997384b8fe7052847ce1e6189af1392d865e6be69835b",
+  "StakingPubKey": "917826e018f056a00b778a58ae83054906957ffd4b6f1b7da083551f7a9f35e02b76ace50424ed7d2c9fc69207a59f0f08a031048f5641db94e77d0648b24d150dedd54bab7cd44b4aa60cfd54be418647b0b3965f8ae54c0bcb48ae9d705162"
+}
+```
+
+### Timing
+
+New nodes are able to join the network each time a new epoch begins.
+An epoch is a period of time (approximately one week) when the node operators in the network are constant.
+At epoch boundaries, newly staked node operators are able to join the network and existing node operators which have unstaked may exit the network.
+You can read more about epochs [here](https://developers.flow.com/nodes/staking/schedule).
+
+
+In order to join the network at epoch N+1, the node **must** be registered with at least 100 FLOW staked prior to the end of epoch N's Staking Auction Phase.
+Currently, an epoch transition on mainnet starts on Wednesday early morning. Hence, the node should be staked by EOD Tuesday to be able to join the network starting on Wednesday.
```

Comment:
@jordanschalm and @joshuahannan - I have explicitly mentioned the day of the week instead of epoch phases here since I wanted to make it easier for new node operators to consume this content. Lemme know what you think.

Reply:
yup! will add that